src/app/footer/footer.component.html

```diff
@@ -1,47 +1,30 @@
-<!-- Footer -->
-<footer class="page-footer font-small bg-dark pt-4">
-    <div class="container-fluid text-center text-md-left">
-      <div class="row">  
-        <!-- Grid column -->
-        <div class="col-md-6 mt-md-0 mt-3">  
-          <h5 class="text-uppercase">Footer Content</h5>
-          <p>Here you can use rows and columns to organize your footer content.</p>  
-        </div>
-        <!-- Grid column -->
-  
-        <hr class="clearfix w-100 d-md-none pb-3">  
-       
-        <div class="col-md-3 mb-md-0 mb-3">
-          <h5 class="text-uppercase">Links</h5>  
-          <ul class="list-unstyled">
-            <li>
-              <a href="#!">Link 1</a>
-            </li>
-            <li>
-              <a href="#!">Link 2</a>
-            </li>
-            <li>
-              <a href="#!">Link 3</a>
-            </li>
-            <li>
-              <a href="#!">Link 4</a>
-            </li>
-          </ul>  
-        </div>
-  
-        <!-- Grid column -->
-        <div class="col-md-3 mb-md-0 mb-3">
-          <h5 class="text-uppercase">Contact US</h5>
-        </div>  
-        <!-- / Grid column -->  
-      </div>      
-    </div>
-  
-    <!-- Copyright -->
-    <div class="footer-copyright text-center py-3">© 2020 Copyright:
-      <a href="#">SLIIT FOSS Community</a>
-    </div>
-    <!-- Copyright -->
-  
-  </footer>
-  <!-- Footer -->
\ No newline at end of file
+<footer class="page-footer font-small pt-4">
+  <div class="content col-6 float-left">
+    <h5 class="text-uppercase">Footer Content</h5>
+    <p>Here you can use rows and columns to organize your footer content.</p>
+  </div>
+  <div class="col-3 float-left">
+    <h5 class="text-uppercase">Links</h5>
+    <ul>
+      <li>
+        <a href="#!">Link 1</a>
+      </li>
+      <li>
+        <a href="#!">Link 2</a>
+      </li>
+      <li>
+        <a href="#!">Link 3</a>
+      </li>
+      <li>
+        <a href="#!">Link 4</a>
+      </li>
```

Comment:
Can you add our social media links with icons

Reply:
Done